runtime.ts

```diff
@@ -204,10 +211,16 @@ export class Runtime {
             this.host.exit(exitCode);
             return null;
           case OpCode.Jump:
-            this.notImplemented('Jump');
+            a = this.readShort();
+            this.pc += a;
             break;
           case OpCode.JumpIfFalse:
-            this.notImplemented('JumpIfFalse');
+            a = this.readShort();
+            b = this.stack.peek();
+
+            if (b === null || falsey(b)) {
```

Comment:
I think we can actually bang "b" - this instruction should *only* occur if the stack has an element on it. Otherwise it's a flagrant error.